CMakeLists.txt

```diff
@@ -112,7 +112,7 @@ endif()
 
 
 
-IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux")
+   IF(${CMAKE_SYSTEM_NAME} MATCHES "Linux" OR ${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
```

Comment:
Thanks for doing this. For future reference it is better to keep pull request according to SRP, i.e. single responsibility principle. here there are multiple targets that you are targeting. Albeit small changes it is still better to split them up.